account_invoice_supplier_self_invoice/views/report_self_invoice.xml

```diff
@@ -3,232 +3,207 @@
     <template id="report_invoice_self_document">
         <t t-set="company" t-value="o.company_id"/>
         <t t-set="o" t-value="o.with_context({'lang':o.partner_id.lang})"/>
-        <div class="header">
-            <div class="row">
-                <div class="col-xs-6" name="company_address">
-                    <address t-field="o.partner_id"
-                             t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
-                    <span t-if="o.partner_id.vat">TIN:
-                        <span t-field="o.partner_id.vat"/>
-                    </span>
+
+        <div class="header" t-att-style="report_header_style">
+            <div class="o_boxed_header">
+            <div class="row mb8">
+                <div class="col-6"/>
```

Comment:
I guess this line can be deleted.

Reply:
This line passes the data from the supplier on the right. I can remove it, but I think it will not look as good.